### ISSUE TITLE
The open-loop detection effect is very poor

### ISSUE THREAD

**Issue by czy1998916** [NVIDIA/Isaac-GR00T#297]
### 🐛 Describe the bug

Thank you for the team's contribution. Let me address my issue. I am using lerobot-s101 and have collected 50 datasets to fine tune the N1.5 model. I have two 24g 4090GPU datasets and trained 10000 steps, but my open-loop evaluation performance is poor. The following image shows one of my open-loop detection results. Video 1 is one of my datasets, and Video 2 is a video of my real-world operation May I ask what caused this and how to solve the problem

<img width="2489" height="1336" alt="Image" src="https://github.com/user-attachments/assets/0f1ee9ca-3126-4f91-abe5-983448bb836a" />

https://github.com/user-attachments/assets/238fee30-4c15-49b3-a1bf-6d3ce5b9c5d5

https://github.com/user-attachments/assets/1b7a0282-93dc-44bf-9321-3033113c08ac

https://github.com/user-attachments/assets/2a9c00c7-0cc2-46fe-9952-550e9d93f5be

### Versions

Thank you for the team's contribution. Let me address my issue. I am using lerobot-s101 and have collected 50 datasets to fine tune the N1.5 model. I have two 24g 4090GPU datasets and trained 10000 steps, but my open-loop evaluation performance is poor. The following image shows one of my open-loop detection results. Video 1 is one of my datasets, and Video 2 is a video of my real-world operation May I ask what caused this and how to solve the problem

**Comment by xwjiang2010** [comment]
If open loop eval result is pool, this feels like a training problem. Can you show your loss curve?

**Comment by czy1998916** [comment]
The problem has been resolved

**Comment by Eric123-tech** [comment]
> The problem has been resolved

Hi there, could you tell me how you solve this problem? when I try to finetune on my new embodiment, both the openloop evaluation and training loss perform badly. Thanks for your response in advance.
